src/write.c

```diff
@@ -1222,6 +1246,9 @@ static avifResult avifEncoderAddImageInternal(avifEncoder * encoder,
                 cellImage = paddedCellImage;
             }
             const int quantizer = item->alpha ? encoder->data->quantizerAlpha : encoder->data->quantizer;
+            // If alpha channel is present, set disableLaggedOutput to AVIF_TRUE. If the encoder supports it, this enables
+            // avifEncoderDataForceKeyframeForAlpha to force a keyframe on the alpha channel if the color channels are a keyframe
```

Comment:
Nit: "if the color channels are a keyframe" should be rephrased. (There is only one color channel.)

How about "whenever the color channel has encoded a keyframe" or "whenever a keyframe has been encoded in the color channel"?

Reply:
Done.